apps/browser/qml/pages/components/PermissionCreateDialog.qml

```diff
@@ -0,0 +1,82 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import Sailfish.WebView.Controls 1.0
+
+Dialog {
+    property alias label: valueButton.label
+    property alias iconSource: valueButton.iconSource
+    readonly property int capability: indexToCapability(valueButton.capability)
+    property alias uri: textField.text
+    readonly property var _regExp: new RegExp("^(http|https):\/\/")
+
+    function indexToCapability(capability) {
+        if (capability === "allow") {
+            return PermissionManager.Allow
+        } else {
+            return PermissionManager.Deny
+        }
+    }
+
+    canAccept: (textField.text !== "" && !textField.errorHighlight)
+
+    DialogHeader {
+        id: header
+
+        //: Accept button text for adding a new permission
+        //% "Add"
+        acceptText: qsTrId("sailfish_browser-he-add")
+    }
+
+    Column {
+        width: parent.width
+        anchors.top: header.bottom
+
+        TextField {
```

Comment:
Did you decide against pre-populating the text entry field? I think you could still add "https://" to it, in order to make it easier for the user.